start-site/src/main/resources/application.yml

```diff
@@ -455,7 +455,7 @@ initializr:
               version: 1.5.1
             - compatibilityRange: "[2.4.1,2.5.0-M1)"
               version: 2.0.1
-            - compatibilityRange: "[2.5.0-M1,2.6.0-M1)"
+            - compatibilityRange: "[2.5.0-M1,2.6.2)"
```

Comment:
Hey @snicoll, I'm trying to put together a quick cron job to test against the latest Spring Boot version (I know this is something you asked us for a while ago, sorry about the delay!)
https://github.com/okta/okta-spring-boot/pull/395

After this gets in and starts running, I'm guessing we could open this up to `[2.5.0-M1,2.8.0-M1)`.
Open to suggestions!

Reply:
Thank you. Do you know what the change above was meant to do?

Reply:
It looks like @arvindkrishnakumar-okta was just trying to mark the compatibility with the current version `2.6.2` (to avoid the warning shown when selecting `okta` from the dependency list).

Reply:
> @arvindkrishnakumar-okta can we please make some progress on this?

@snicoll apologies for the delay on this one.

Reply:
@snicoll The idea of @bdemers 's PR https://github.com/okta/okta-spring-boot/pull/395 is to test our Okta Springboot SDK version against the latest milestone release of Springboot.